src/components/BookCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBooks } from '../redux/books/books';
+
+const BookCard = (props) => {
+  const dispatch = useDispatch();
+  const { id, title, author } = props;
+  return (
+    <li className="">
+      <div className="px-6 py-4 m-1">
+        {title}
+        -
+        {author}
+      </div>
+      <button
+        type="button"
+        onClick={() => {
+          dispatch(removeBooks(id));
+        }}
+      >
+        Remove
+      </button>
```

Comment:
- [ ] [OPTIONAL] Great implementation @jerryowusu I will advise you style your `check status` and `remove` buttons properly just like your styled the `add book` button so it looksmore professional.

Reply:
Thank you for the advice @Whoistolu, I will work on that.